ui/webui/webui-desktop

```diff
@@ -86,4 +86,9 @@ else
     sleep 3
 fi
 
-HOME="$BROWSER_HOME" $BROWSER http://127.0.0.90"$URL_PATH"
+trap 'kill $B_PID' INT TERM EXIT
```

Comment:
Oh, there's already a trap in line 76. I'm not actually sure if they are additive, or the second one would replace the first (my hunch is the latter?) Can you just move this kill into line 76 then?

Reply:
So I finally modified it to just one trap and created a cleanup function to make the code clearer.